stylesheets/stylesheets/popup.css

```diff
@@ -0,0 +1,161 @@
+.mobilehide {
+  display: none;
+}
+
+.popup {
+  width: 100%;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.9);
+  z-index: 10;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: nowrap;
+  position: fixed;
+  margin-top: 50px;
+  overflow: scroll;
+}
```

Comment:
[OPTIONAL] Also here please make sure the project is rendering correctly. :eyes: :eyes:. Kindly fix the source path of the image to make it render properly.   
![image](https://user-images.githubusercontent.com/98363075/193582902-c566e700-00de-4598-97aa-fc654367bba4.png)